lib/course/middleware/assets.rb

```diff
@@ -57,6 +57,15 @@ def full_path(env)
       def danger_path?(env)
         env["REQUEST_PATH"].sub(PUBLIC_URL, "").include?("../")
       end
+
+      def headers(env)
+        header = {}
+        header["content-type"] = env["CONTENT_TYPE"] || "text/plain"
```

Comment:
I'm not sure, why you want to specify a content type here, but for assets, which are usually files css/js/png/pdf/jpg/csv/blah_blah, text/plain would not be the best default value.